Classes/Common/Document.php

```diff
@@ -48,6 +49,14 @@
  */
 abstract class Document
 {
+    /**
+     * This holds the logger
+     *
+     * @var LogManager
```

Comment:
Good idea, but would you draw the changes to LogManager to a separate pull request? This is not related to the ocr highlighting.